### DISCUSSION TITLE
V2 窗口不能拉大小.有些看不到.

### DISCUSSION THREAD

**Discussion by hqwl1985** [cangzhang/ChampR#240] [Ideas]
V2 窗口不能拉大小.有些看不到.能和三个版本一样拉大小就完美了

**Comment by cangzhang** [comment]
你的显示器分辨率是多少? 发个截图看看效果

**Reply by hqwl1985** [reply]
5120 x 1440 老版本可以调大小.新版本不能调整.

**Comment by hqwl1985** [comment]
&nbsp;


13774366
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Al ***@***.***&gt;; 
发送时间: 2022年11月14日(星期一) 晚上7:43
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [cangzhang/champ-r] V2 窗口不能拉大小.有些看不到. (Discussion #240)





 
你的显示器分辨率是多少? 发个截图看看效果
